pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -6691,6 +6694,10 @@ def test_redundancy_contract_expand_nblts_not_nbls_times_ntimes(method, casa_uvf
         with uvtest.check_warnings(
             UserWarning,
             [
+                "The uvw_array does not match the expected values given the antenna "
+                "positions."
+            ]
```

Comment:
If the problem was that there are now two of these instead of one, you could just multiply the list that matches this below by 2.

Reply:
Ahh - for some reason I thought it was order sensitive. Fixed in 5279a6f.